Games/types/Mafia/roles/cards/CleanseVisitors.js

```diff
@@ -19,12 +20,25 @@ module.exports = class CleanseWerewolfVisitors extends Card {
                     for (let action of this.game.actions[0]) {
                         if (
                             action.target == this.actor &&
-                            action.actor.hasEffect("Werewolf") &&
                             action.priority > this.priority &&
                             !action.hasLabel("hidden")
                         ) {
-                            action.actor.removeEffect("Werewolf", true);
-                            cleansedWolves[action.actor.id] = true;
+                            if (action.actor.hasEffect("Werewolf")){
+                                action.actor.removeEffect("Werewolf", true);
+                                cleansedWolves[action.actor.id] = true;
+                            }
+                            if (action.hasLabel("kill") &&
+                            !action.hasLabel("werewolf") &&
+                            action.actor.role.name !== "Lycan" &&
+                            Random.randInt(0, 1)){
+                                if (action.actor.role.alignment === "Village"){
+                                    action.actor.setRole("Villager");
+                                }else if(action.actor.role.alignment === "Mafia"){
+                                    action.actor.setRole("Traitor");
+                                }else{
```

Comment:
Should there be a case for the Monsters alignment? Otherwise it looks fine to be merged.

Reply:
Monster case could be added when more monster roles are created, right now it should make survivors from monsters.